webpack/global_index.js

```diff
@@ -43,3 +44,4 @@ addGlobalFill(
 );
 addGlobalFill('details-cards', 'Installable errata', <ErrataOverviewCard key="errata-overview" />, 1900);
 addGlobalFill('host-tab-details-cards', 'Installed products', <InstalledProductsCard key="installed-products" />, 100);
+addGlobalFill('host-tab-details-cards', 'Registration details', <RegistrationCard key="registration-details" />, 100);
```

Comment:
```suggestion
addGlobalFill('host-tab-details-cards', 'Registration details', <RegistrationCard key="registration-details" />, 200);
```
As per the mockup, Registration details should appear before Installed products, so let's increase the weight